src/commands/mod.rs

```diff
@@ -81,7 +83,13 @@ pub fn run(
                 args.quiet,
                 question_policy,
                 file_visibility_policy,
-                level,
+                if fast {
+                    Some(0)
+                } else if slow {
+                    Some(i16::MAX)
+                } else {
+                    level
+                },
```

Comment:
Can you bind this expression to a variable? Nested expressions are one of my strong pet peeves xD, I like giving names because `Some(0)` doesn't have a clear meaning.